tests/test_parse.py

```diff
@@ -273,9 +273,9 @@ def test_set_ethics_bad_data(self, fake_get_ethics):
 
     def test_parse_ethics(self):
         "test examples of parsing ethics data"
-        with open(fixture_path('ethic_data.txt'), 'rb') as open_file:
+        with open(fixture_path('ethic_data.txt'), 'r') as open_file:
```

Comment:
Could just omit the `'r'` now as its the default mode anyway.

Reply:
:+1: